packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -31,9 +31,22 @@ import 'theme.dart';
 ///
 /// {@tool dartpad}
 ///
-///
 /// ** See code in examples/api/lib/material/reorderable_list/reorderable_list_view.0.dart **
-///{@end-tool}
+/// {@end-tool}
+///
+/// Default [proxyDecorator] can overriden to customize an item when it is being dragged.
+/// {@tool snippet}
+///
+/// Here item's background color and shadow color customized compared to default behavior
```

Comment:
While a drag is underway, the widget returned by the [proxyDecorator] serves as a "proxy" (a substitute) for the item in the list. The proxy is created with the original list item as its child. The [proxyDecorator] in this example is similar to the default one except that it changes the proxy item's background color.

Reply:
That is an awesome description, thanks!